lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -164,16 +163,6 @@ def trigger(*args)
       run_async(*args)
     end
 
-    def detect_capabilities
```

Comment:
Doesn't it revert https://github.com/theforeman/smart_proxy_remote_execution_ssh/pull/115/files ?

Reply:
Not really, it just got moved around a little bit. The unshare detection now happens here https://github.com/theforeman/smart_proxy_remote_execution_ssh/pull/116/files#diff-f7a31028d1cbfee3429cdcb724c2fd35effcf3b665e39aec9c348ee015415313R196-R199

Reply:
done